src/graphics/program-lib/programs/standard.js

```diff
@@ -1567,6 +1571,15 @@ var standard = {
             }
         }
 
+        if (options.opacityFadesSpecular === false) {
+            if (options.blendType === BLEND_NORMAL || options.blendType === BLEND_PREMULTIPLIED) {
+                code += "float specLum = dot((dSpecularLight + dReflection.rgb * dReflection.a) * dSpecularity, vec3( 0.2126, 0.7152, 0.0722 ));\n";
```

Comment:
Can this be moved into a shader chunk?

Reply:
I thought about putting into a chunk but decided against it because the javascript logic isolates a single glsl line - and it seems more readable in this form to me.

Reply:
The question is, will other developers will want to override its behaviour, or this will *never* be the case?